src/lib/bootloader/write_dialog.rb

```diff
@@ -9,11 +9,13 @@ class WriteDialog
     include Yast::I18n
 
     # Write settings dialog
-    # @return `:abort` if aborted and `:next` otherwise
+    #
+    # @return [Symbol] :abort if aborted
+    #                  :next otherwise
     def run
       Yast::Wizard.RestoreHelp(help_text)
-      ret = Yast::Bootloader.Write
-      ret ? :next : :abort
+      return :next if Yast::Bootloader.Write
+      :abort
```

Comment:
In fact, I though first in your proposed improvement, `Yast::Bootloader.Write ? :next : :abort`, but usually I try to avoid the ternary operator. Moreover, the use of "guard clauses" if looks more readable and natural to me.

That said, if you feel uncomfortable with this style I can to use your suggestion instead.

Reply:
well, for this case I do not like much guard clauses as it hides that real action is done behind if at the end of line, which is not nature for me.
If you want to use guard clause then I prefer something like
```
failed = !Yast::Bootloader.Write
return :abort if failed

:next
```
including new line so it is clear separation.

Reply:
I (partially) agree with 

>  hides that real action is done behind if at the end of line

So, I am going to use the ternary operator for this case, in order to keep it simple and not add extra lines only for use an specific technique.

Thank you.